pandas/io/parsers.py

```diff
@@ -348,38 +347,47 @@ def _validate_nrows(nrows):
     return nrows
 
 
+_compression_to_extension = {
```

Comment:
> you prob need to rebuild the extensions.

`python setup.py build_ext --inplace` did the trick!

https://github.com/pandas-dev/pandas/pull/14576/commits/a821b8207ea3a8ed54b3a79a7ae208da2adefc44 moves `_infer_compression` to `io/common.py`.